utils/mil_tools/scripts/ReadMe.txt

```diff
@@ -0,0 +1,9 @@
+sProgramFinder:
```

Comment:
Looks like you got a typo.
Also, you can use markdown to make the readme look nicer.
Convientially the file name should be ```readme.md``` or ```README.md```